README.md

```diff
@@ -11,6 +11,30 @@
 - [Run a SignalFlow computation](#run-a-signalflow-computation)
 - [License](#license)
 
+SignalFlow is the SignalFx real-time analytics computation language. The SignalFlow
+API allows SignalFx users to execute real-time streaming analytics computations on
+the SignalFx platform. For more information, see the Splunk Observability Cloud
+developer documentation:
+
+* [SignalFlow Overview](https://dev.splunk.com/observability/docs/signalflow/)
+* [SignalFlow API Reference](https://dev.splunk.com/observability/reference/api/signalflow/latest)
+
+The SignalFlow Python library is a client you can use to send SignalFlow programs
+to the SignalFx back end. The client opens a connection to the back end and sends
```

Comment:
```suggestion
to the SignalFx back end. The client opens a connection to the back end and receives
```

Reply:
It sends the query (or program) and receives a stream of information thereafter.